public/script.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * @file script.js Главный файл скрипта, содержащий логику для получения и отображения популярных треков и артистов.
+ */
+/**
+ * Получает список популярных треков с Last.fm.
+ *
+ * @async
+ * @function getPopularTracks
+ * @returns {Promise<Array<object>>} Promise, разрешающийся массивом объектов треков. Возвращает пустой массив в случае ошибки.
+ */
+async function getPopularTracks() {
+  const url = `http://ws.audioscrobbler.com/2.0/?method=chart.getTopTracks&api_key=2467e45594a2085cd6fbcdb081b93c5f&format=json`;
```

Comment:
лучше всегда использовать https, т.к. браузер может блокировать незашифроанные запросы из-за, встроенных политик 
если не уверен в протоколе можно воспользоваться подобным маленьким лайфхаком (не для продакшена):

```suggestion
  const url = `//ws.audioscrobbler.com/2.0/?method=chart.getTopTracks&api_key=2467e45594a2085cd6fbcdb081b93c5f&format=json`;
```

оставить два слеша, там дальше само определится